test/meson.build

```diff
@@ -68,4 +68,9 @@ simde_include_dir = include_directories('..')
 
 subdir('common')
 subdir('x86')
-subdir('arm')
+
+# Currently it is nearly impossible to support non-native ARM on elbrus.
+# It's compiler bug, so it will be possible in future versions.
+if cc.get_id() != 'lcc'
+    subdir('arm')
+endif
```

Comment:
Ouch.  I don't suppose it's an issue we can work around in the code?  Can you provide a few more details about what exactly the problem is, and if possible the relevant bug number or a link to the relevant bug in the issue tracker?

Generally I'd feel better about moving this logic into the code (see the `SIMDE_BUG_*` macros in simde-common.h), but depending on what the bug is that may not be feasible.

Reply:
There is a ton of errors like this:
```
lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 202: error #415:
          no suitable constructor exists to convert from "__m128" to
          "simde_float32x4_private"
      return _mm_loadu_ps(ptr);
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 222: error #415:
          no suitable constructor exists to convert from "__m128d" to
          "simde_float64x2_private"
      return _mm_loadu_pd(ptr);
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 242: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_int8x16_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 262: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_int16x8_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 282: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_int32x4_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 302: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_int64x2_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 322: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_uint8x16_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 342: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_uint16x8_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 362: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_uint32x4_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
             ^

lcc: "../../auto/patchouli/src/simde/test/arm/neon/../../../simde/arm/neon/ld1.h", line 382: error #415:
          no suitable constructor exists to convert from "__m128i" to
          "simde_uint64x2_private"
      return _mm_loadu_si128(SIMDE_ALIGN_CAST(const __m128i*, ptr));
```
And they are virtually in every file inside `arm` subdirectory. Probably it could be solved, but for now I have no idea how to deal with such a massive amount of errors. But if this really matter, I'm gonna check how that can be solved.

Reply:
That… shouldn't be happening.  My guess would be there is some ifdef on `SIMDE_ARCH_X86` in neon/types.h.  I'll try to take a look at it once I can ssh into an e2k machine.

Reply:
Wow, that was a very useful gudeance! It ended up in fixing just a single line in f3fe103 rather that rewriting every single file inside `simde/arm` as I expected before.